### PR TITLE
kill two todos in deleteContainer operation

### PR DESCRIPTION
**Pull request by bergwolf**
Also clean up glog trailing newline a bit in cli/container.go.

### CODE REVIEW

File: cli/container.go [hunk 1]
```diff
@@ -145,13 +145,18 @@ func createContainer(options runvOptions, vm *hypervisor.Vm, container, bundle,
...
-	exitedHost := syscall.Kill(state.Pid, syscall.Signal(0)) != nil
+	exitedVM := true
+	for _, c := range vm.ContainerList() {
+		if c == container {
+			exitedVM = vm.SignalProcess(container, "init", syscall.Signal(0)) != nil // todo: is this check reliable?
+			break
+		}
+	}
+	exitedHost := true
+	if containerShimAlive(state) {
```
Comment (laijs) [inline cli/container.go]: Is system.Stat() more reliable than syscall.Kill() in which case we don't need syscall.Kill() anymore?

Comment (bergwolf) [inline cli/container.go]: Good point! Let's replace syscall.Kill() with system.Stat() for shim process checking.

**Comment by bergwolf** [comment]
@laijs updated.

**Comment by laijs** [comment]
LGTM

**Pull request merged**